### PR TITLE
NO JIRA Implement Search feature

### PR DESCRIPTION
**Pull request by fayloads**
As a user of Anemone, one can now query their personal store in Anemone.
The endpoint as have been added to our Postman Collection
- `/search` is a dashboard view of the search feat
- `/search/autocomplete?q=` is the endpoint that will search the database associated with your account
There is an example in the Postman Collection showing how to build a simple query. At this point, you can search by name or file type, whatever is in the record's path name attribute will be found.

Screen Shot 2022-04-24 at 10 03 07 PM

### CODE REVIEW

File: project-anemone/routes/web.php [hunk 1]
```diff
@@ -25,3 +24,9 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::group([SearchController::class],function(){
+    Route::get('search', [SearchController::class, 'index']);
+    Route::get('autocomplete', [SearchController::class, 'autocomplete'])->name('autocomplete');
+    
+});
```
Comment (ApplebaumIan) [inline project-anemone/routes/web.php]: This should probably be a part of the `api.php` as at the moment on the EBP server any of the API calls are only accessible by https://anemone.avalonbloom.com/api/. All calls to https://anemone.avalonbloom.com/ are routed to the React App. The other thing that may need to be decided on is whether a user should be authenticated in order to search. My vote is that you would.

**Review by ApplebaumIan — CHANGES_REQUESTED** [review]
I'm also getting a 500 error on the search page and autocomplete query for some reason.

File: project-anemone/routes/web.php (hunk 1 above)
Comment (fayloads) [reply project-anemone/routes/web.php]: fair point! I have moved the endpoints behind API.

File: project-anemone/routes/console.php [hunk 2]
```diff
@@ -16,4 +16,4 @@
 
 Artisan::command('inspire', function () {
     $this->comment(Inspiring::quote());
-})->purpose('Display an inspiring quote');
+})->purpose('2b V !2b');
```
Comment (ApplebaumIan) [inline project-anemone/routes/console.php:19]: Bruhhhh 🤣

File: README.md [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
-Our software is primarily designed to help people who are interested in collection information. The utmost intention of the software is to make it easy for people to query the web without going down rabbit holes. Users can create personal accounts through which they can store information, and log in to the same account to display historically stored information if the device is changed. 
+Anemone is an application that centralizes users' documentation queries. 
+A user will be able to collect and retrieve references concurrently with various active applications. 
+In other words we want Anemone to coexist while a user is working in any other application.
+Our software is primarily designed to help people who are interested in collecting information. 
+The utmost intention of the software is to assist efficient and effective experiences for people to query personal results. 
+Users can create personal accounts through which they can store information and display historically stored information if the device is changed.
```
Comment (ApplebaumIan) [inline README.md:9]: This last sentence is a bit confusing.

**Review by ApplebaumIan — APPROVED** [review]
Code looks good and works for me! The README last sentence is a bit confusing.

**Pull request merged**